### PR TITLE
[stable/sentry] Add option to disable dbInit job hook

### PR DESCRIPTION
**Pull request by max-rocket-internet**
#### What this PR does / why we need it:

Adds an option to disable dbInit job hook.

I don't know about everyone else but every time I sync this chart it creates this job that always fails and it's quite annoying.

#### Special notes for your reviewer:

None

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

**Comment by okgolove** [comment]
Hi @max-rocket-internet! Nice to see terraform-aws-eks module maintainer here :)
Could you show error log from dbinit job?

**Comment by max-rocket-internet** [comment]
Hey @okgolove,
I get this:

```
Upgrading release=sentry, chart=stable/sentry
UPGRADE FAILED
Error: no kind "Job" is registered for version "batch/v1" in scheme "k8s.io/kubernetes/pkg/api/legacyscheme/scheme.go:30"


FAILED RELEASES:
NAME
sentry
in kubernetes/helmfiles/main.yaml: in .helmfiles[6]: in clusters/eu01-infra01.yaml: failed processing release sentry: helm exited with status 1:
  Error: UPGRADE FAILED: no kind "Job" is registered for version "batch/v1" in scheme "k8s.io/kubernetes/pkg/api/legacyscheme/scheme.go:30"
```

Pod log:

```
$ kubectl logs sentry-db-init-4n7sr
Traceback (most recent call last):
  File "/usr/local/bin/sentry", line 10, in <module>
    sys.exit(main())
  File "/usr/local/lib/python2.7/site-packages/sentry/runner/__init__.py", line 162, in main
    cli(prog_name=get_prog(), obj={}, max_content_width=100)
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/sentry/runner/decorators.py", line 35, in inner
    configure()
  File "/usr/local/lib/python2.7/site-packages/sentry/runner/__init__.py", line 125, in configure
    configure(ctx, py, yaml, skip_service_validation)
  File "/usr/local/lib/python2.7/site-packages/sentry/runner/settings.py", line 149, in configure
    from .initializer import initialize_app, on_configure
  File "/usr/local/lib/python2.7/site-packages/sentry/runner/initializer.py", line 17, in <module>
    from sentry.utils.sdk import configure_sdk
  File "/usr/local/lib/python2.7/site-packages/sentry/utils/sdk.py", line 20, in <module>
    from sentry.utils import metrics
  File "/usr/local/lib/python2.7/site-packages/sentry/utils/metrics.py", line 26, in <module>
    backend = get_default_backend()
  File "/usr/local/lib/python2.7/site-packages/sentry/utils/metrics.py", line 23, in get_default_backend
    return cls(**settings.SENTRY_METRICS_OPTIONS)
  File "/usr/local/lib/python2.7/site-packages/sentry/metrics/statsd.py", line 12, in __init__
    self.client = statsd.StatsClient(host=host, port=port)
  File "/usr/local/lib/python2.7/site-packages/statsd/client.py", line 138, in __init__
    host, port, 0, socket.SOCK_DGRAM
socket.gaierror: [Errno -2] Name or service not known
```

**Comment by max-rocket-internet** [comment]
And values for db are this:

```yaml
postgresql:
  enabled: false
  postgresHost: xxxxx.zzzzzzz.eu-west-1.rds.amazonaws.com
  postgresPort: 5432
```

**Comment by max-rocket-internet** [comment]
@okgolove I'm not sure what to make of the e2e test failures?

```
ERROR:  relation "sentry_option" does not exist at character 114
```

**Comment by okgolove** [comment]
Looks strange.
I should try to reproduce the same situation.

**Comment by max-rocket-internet** [comment]
Any update @okgolove?

**Comment by okgolove** [comment]
I also get 
```Error: unable to recognize "": no matches for kind "Job" in version ""
```

Still investigating.

**Comment by max-rocket-internet** [comment]
Any update @okgolove?

**Comment by okgolove** [comment]
@max-rocket-internet sorry for delay, I'm pretty busy now.
The strange thing is I can't reproduce it locally!

**Comment by okgolove** [comment]
It looks as if the e2e tests are broken.
I've wrote to a charts slack channel.
Hope it will be fixed.

File: stable/sentry/templates/hooks/db-init.job.yaml [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # https://docs.sentry.io/server/installation/docker/#running-migrations
+{{- if .Values.hooks.dbInit.enabled -}}
```
Comment (okgolove) [inline stable/sentry/templates/hooks/db-init.job.yaml]: @max-rocket-internet could you remove the second one dash?
It seems it brokes template:
```
---
# Source: sentry/templates/hooks/db-init.job.yaml
# https://docs.sentry.io/server/installation/docker/#running-migrationsapiVersion: batch/v1
kind: Job
```

https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/helm_charts/20293/pull-charts-e2e/1246833451739385856/#1:build-log.txt%3A202

Comment (max-rocket-internet) [reply stable/sentry/templates/hooks/db-init.job.yaml]: OK done!

**Comment by max-rocket-internet** [comment]
Looks good now @okgolove, thank you 🙂

**Pull request merged**